src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -66,123 +62,117 @@ var fluid_2_0 = fluid_2_0 || {};
         sliderOptions: {
             orientation: "horizontal",
             step: 1.0
-        },
-        invokers: {
-            refreshView: {
-                funcName: "fluid.textfieldSlider.refreshView",
-                args: ["{that}"]
-            }
-        },
-        finalInitFunction: "fluid.textfieldSlider.finalInit",
-        renderOnInit: true
-    });
-
-    fluid.textfieldSlider.finalInit = function (that) {
-
-        that.applier.modelChanged.addListener("value", function (newModel) {
-            that.events.modelChanged.fire(newModel.value);
-        });
-
-        if (that.options.renderOnInit) {
-            that.refreshView();
         }
-    };
-
-    fluid.textfieldSlider.refreshView = function (that) {
-        that.textfield.container.val(that.model.value);
-        that.events.afterRender.fire(that);
-    };
+    });
 
     fluid.defaults("fluid.textfieldSlider.textfield", {
-        gradeNames: ["fluid.viewComponent", "autoInit"],
+        gradeNames: ["fluid.viewRelayComponent", "autoInit"],
+        range: {}, // should be used to specify the min, max range e.g. {min: 0, max: 100}
+        modelRelay: {
+            target: "value",
+            singleTransform: {
+                type: "fluid.transforms.stringToNumber",
+                input: "{that}.model.stringValue"
+            }
+        },
+        modelListeners: {
+            value: {
+                "this": "{that}.container",
+                "method": "val",
+                args: ["{change}.value"]
+            },
+            // When the input field receives a invalid input such as a string value, ignore it and populate the field with the previous value
+            stringValue: {
```

Comment:
Yes, namespaces are currently ignored in databinding. However we should leave a comment here to explain the problems that might arise with this implementation.